File: sistema_mercado_python/modelos/Produto.py
class Produto:
    def __init__(self, nome, preco):
        self._nome = nome
        self._preco = preco

    def get_preco(self):
        return self._preco

    def get_nome(self):
        return self._nome

    def set_nome(self, nome):
        self._nome = nome

    def get_preco(self):
        return self._preco

    def set_preco(self, preco):
        self._preco = preco

File: sistema_mercado_java/src/modelo/Produto.java
package modelo;

public class Produto implements Exibivel {
    private String nome;
    private double preco;

    public Produto(String nome, double preco) {
        this.nome = nome;
        this.preco = preco;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public double getPreco() {
        return preco;
    }

    public void setPreco(double preco) {
        this.preco = preco;
    }

    @Override
    public String exibirDados() {
        String retorno = "Dados do produto:\n";
        retorno += "Produto" + nome + "\n";
        retorno += "Preco" + preco + "\n";
        return retorno;
    }

    @Override
    public String toString() {
        return "Produto: " + nome + ", Preço: " + preco;
    }
}